src/components/SettingsModal/SettingsModal.vue

```diff
@@ -353,6 +358,7 @@ export default {
           ).toNumber()
         );
       } else {
+        if (!new BigNumber(this.customGas).gte(1)) return;
```

Comment:
this shouldn't be necessary after the above change

Reply:
made the change